src/components/theme-switcher/theme-switcher.module.css

```diff
@@ -1,7 +1,7 @@
 .root {
 	position: relative;
 	color: var(--token-color-foreground-primary);
-	width: 100%;
+	width: fit-content;
```

Comment:
Made this update so tablet doesn't look so awkward 
<img width="1029" alt="Screenshot 2023-04-03 at 1 41 01 PM" src="https://user-images.githubusercontent.com/36613477/229652424-2d956e77-345d-463e-977e-613f6ff251c1.png">